decidim-core/lib/decidim/has_category.rb

```diff
@@ -13,6 +13,12 @@ module HasCategory
 
       validate :category_belongs_to_organization
 
+      def previous_category
+        return if self.categorization.versions.count <= 1
+        category_id = self.categorization.paper_trail.previous_version.decidim_category_id
+        Decidim::Category.find_by_id category_id
```

Comment:
Memoize this

Reply:
Does something like this work?

`categorization.previous_version.category`

Reply:
`categorization.previous_version.category` doesn't work as it returns a`PaperTrail` object
  
line 18 and 19 can be replaced by:
```
Decidim::Category.find_by_id(self.categorization.versions.last.reify.decidim_category_id)
```

is this better?

Reply:
Seems ok!